CacheClearer/Opcache.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/**
+ * Created by PhpStorm.
+ * User: Yuriy Maksimenko <yuriy.maksimenko@tonicforhealth.com>
+ * Date: 3/21/16
+ * Time: 2:17 PM
+ */
+namespace CacheTool\Bundle\CacheClearer;
+
+use CacheTool\CacheTool;
+
+/**
+ * Class Opcache
+ */
+class Opcache implements ClearerInterface
+{
+    /**
+     * @var CacheTool
+     */
+    protected $cacheTool;
+
+    /**
+     * Opcache constructor.
+     *
+     * @param CacheTool $cacheTool
+     * @param bool      $opcacheEnabled
+     */
+    public function __construct(CacheTool $cacheTool, $opcacheEnabled)
+    {
+        $this->cacheTool = $cacheTool;
+        $this->opcacheEnabled = $opcacheEnabled;
+    }
+
+    /**
+     * Check if cache enabled
+     *
+     * @return bool
+     */
+    public function isEnabled()
+    {
+        return ($this->opcacheEnabled && extension_loaded('Zend OPcache') && ini_get('opcache.enable') === '1') ? true : false;
```

Comment:
Because you're running this in the console, which might not have opcache enable whereas the server could have it enabled.